app/scripts/modules/core/src/modal/modals.less

```diff
@@ -271,15 +271,27 @@ html .select2-container-multi {
   }
 }
 
+.overlay-modal-appear {
```

Comment:
hmm this makes me wonder again if we could just use @erikmunson 's new modal code with all of its whirley twirley anim bits

Reply:
this is the weirdo thing that drops over the existing modal for the task progress — not the modal itself.